doc/interop-test-descriptions.md

```diff
@@ -1005,14 +1005,21 @@ Client asserts:
 ### rpc_soak
 
 The client performs many large_unary RPCs in sequence over the same channel.
-The client records the latency and status of each RPC in some data structure.
-If the test ever consumes `soak_overall_timeout_seconds` seconds and still hasn't
-completed `soak_iterations` RPCs, then the test should discontinue sending RPCs
-as soon as possible. After performing all RPCs, the test should examine
-previously recorded RPC latency and status results in a second pass and fail if
-either:
+The total number of RPCs to execute is controlled by the `soak_iterations` 
+parameter, which defaults to 10. The number of threads used to execute RPCs 
+is controlled by `soak_num_threads`. By default, `soak_num_threads` is set to 1. 
```

Comment:
The first sentence says that it does "RPCs in sequence". This isn't modifying it at all, so there is only one RPC concurrently?

Reply:
Although the sentence mentions that the RPCs are executed 'in sequence,' it refers to how each individual thread handles the RPCs. In this case, each thread processes its RPCs using blocking api, meaning that within a single thread, the RPCs are executed sequentially. However, since multiple threads might be available (based on the soak_num_threads parameter), the system can execute multiple RPCs concurrently. So, while RPCs run in sequence within a thread, there can be multiple threads running their own sequences of RPCs concurrently, allowing parallel execution of the overall workload.

Reply:
> it refers to how each individual thread handles the RPCs

Does it? How should the reader know that?

Reply:
Updated!

Reply:
Eric, could you help me merge this PR? Thanks!

Reply:
I can't, actually.

Reply:
Do you know who has the authorization to merge it? I don't have permission to do so.